### ISSUE TITLE
NullPointerException when chaining

### ISSUE THREAD

**Issue by JerryShea** [OpenHFT/Chronicle-Wire#224]
```
7080 [main] WARN xxxxxxx.ServiceContainerTEXTInterceptingMethodReader - Failure to dispatch message, will retry to process without generated code: heartbeat()
java.lang.NullPointerException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.openhft.chronicle.wire.AbstractGeneratedMethodReader.actualInvoke(AbstractGeneratedMethodReader.java:174)
	at software.chronicle.services.util.EventMethodReaderInterceptor.intercept(EventMethodReaderInterceptor.java:33)
	at net.openhft.chronicle.wire.TextMethodTester.lambda$run$1(TextMethodTester.java:170)
	at xxxxxxx.ServiceContainerTEXTInterceptingMethodReader.readOneCall(ServiceContainerTEXTInterceptingMethodReader.java:429)
	at net.openhft.chronicle.wire.AbstractGeneratedMethodReader.readOne0(AbstractGeneratedMethodReader.java:83)
	at net.openhft.chronicle.wire.AbstractGeneratedMethodReader.readOne(AbstractGeneratedMethodReader.java:108)
	at net.openhft.chronicle.wire.TextMethodTester.run(TextMethodTester.java:182)
	at software.chronicle.services.YamlTester.testMessages(YamlTester.java:45)
	at software.chronicle.services.YamlTester.testMessages(YamlTester.java:26)

```

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.84](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.84), [BOM-2.20.123](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.123)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
